### PR TITLE
Update README.md - in Windows msys2 instructions, specify that PATH needs to be updated

### PR DESCRIPTION
**Pull request by UrKr**
I had problems getting rgtk to build on Windows. I eventually resolved it by adding C:\msys32\mingw32\bin to the path. I think it would be useful to add that to the instructions under msys2 for windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ or follow these steps:
  ```Shell
  $ pacman -S mingw-w64-x86_64-gtk3
  ```
+ 
+Make sure that either `<your msys installation folder>\mingw32\bin` or `<your msys installation folder>msys32\mingw64\bin` is in your `PATH` e.g. (if you have msys32 installed at `c:\msys32`, add `c:\msys32\mingw32\bin` or `c:\msys32\mingw64\bin` to your `PATH`)
```
Comment (GuillaumeGomez) [inline README.md]: You forgot an ending '.'. ;)

**Comment by GuillaumeGomez** [comment]
cc @gkoz

**Comment by gkoz** [comment]
This recipe troubles me. As noted at https://github.com/gtk-rs/gtk/pull/218#discussion_r49073417 I have doubts that using gcc from msys2 will result in successful builds. I've had failures to find some pthread symbols on appveyor. How is my use different from yours?

**Comment by UrKr** [comment]
Hm, you know this is actually a bit over my head. All I really know is, that adding that to the path made it work for me. (win7 64bit, rust GNU nightly 1.7 32bit). 

I did notice that every other build attempt seems to result in "unknown error". So sometimes it builds, and sometimes it fails. When it builds, the resulting .exe works. I have no idea why this would be. 

Also @GuillaumeGomez I'll add the dot if I can figure out how to edit pull requests. Also, I hope you mean a dot after the last ")". Haha, sorry, I'm terrible at this.

File: README.md (hunk 1 above)
Comment (gkoz) [inline README.md]: > `<your msys installation folder>msys32\mingw64\bin`

It fells like the presence of `msys32` there is a typo.

**Comment by GuillaumeGomez** [comment]
@UrKr: On the web version, it's not easy. Here are the steps:
* go to your repository, pick branch `patch-1`
* edit this file

Now the "hard" part:
* you'll need a local copy of your repository (download github software)
* then, if you can find the console (otherwise, I can't help you), enter:
```Shell
git checkout patch-1 && git rebase -i HEAD~2
``` 
* a file editor should ask to edit things and looks like this:

```Shell
pick 9cc83b4 Add the missing line
pick 11e0e6d Add libsecret and libvte

# Rebase f32e511..11e0e6d onto f32e511
#
# Commands:
#  p, pick = use commit
#  r, reword = use commit, but edit the commit message
#  e, edit = use commit, but stop for amending
#  s, squash = use commit, but meld into previous commit
#  f, fixup = like "squash", but discard this commit's log message
#  x, exec = run command (the rest of the line) using shell
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#
# Note that empty commits are commented out
```

Replace the second "pick" by "s" (so here you would replace "pick 11e0e6d Add libsecret and libvte" by "s 11e0e6d Add libsecret and libvte").

* then add "#" to comment the commit.
* then:
```Shell
git push -f origin HEAD:patch-1
```

For more information, look for "squash git commit" on the internet.

**Comment by gkoz** [comment]
> Hm, you know this is actually a bit over my head. All I really know is, that adding that to the path made it work for me. (win7 64bit, rust GNU nightly 1.7 32bit). 

Okay I guess merging this clarification won't hurt.

**Comment by GuillaumeGomez** [comment]
But before, we'll have to wait you to squash your commits @UrKr.

**Comment by GuillaumeGomez** [comment]
#222

**Comment by UrKr** [comment]
I made a fork before making a pull-request, so my fork ended up being different than the one in the pull-request (it wasn't showing my edit). I deleted it, because I though it wasn't this one, but it was. Now it's gone, and I don't know what I'm supposed to do from here, because I think this pull-request was dependent on this repository. 

I really don't know git-hub from a contributor standpoint (or how to use it). I assumed it was the etiquette for me to edit the readme, since I posted the issue, but I suppose you were offering in case I wanted it on my github record? I don't know. Anyway, I probably made a bit of a mess of this. If it's not too much trouble you might as well edit the readme yourself if you think it should be added. 

Sorry for wasting your time. :/ 

This was the edit, in case you need it: 
"Make sure that either `<your msys installation folder>\mingw32\bin` or `<your msys installation folder>\mingw64\bin` is in your `PATH` e.g. (if you have msys32 installed at `c:\msys32`, add `c:\msys32\mingw32\bin` or `c:\msys32\mingw64\bin` to your `PATH`)."

**Comment by GuillaumeGomez** [comment]
@UrKr: If you really don't know how to do, I can cherry-pick your contribution and merge it myself (so the commit will be yours).

**Comment by UrKr** [comment]
Well, I could probably figure it out, but I have to be doing something else right now, so I'm kind of in a rush. I don't mind if the commit isn't mine.

**Comment by GuillaumeGomez** [comment]
I do mind that the contributions are linked to their owners. Don't worry then, I cherry pick.

**Comment by GuillaumeGomez** [comment]
@UrKr: Done.

**Comment by UrKr** [comment]
@GuillaumeGomez  Ok, thanks!

**Comment by UrKr** [comment]
"I did notice that every other build attempt seems to result in "unknown error". So sometimes it builds, and sometimes it fails. When it builds, the resulting .exe works. I have no idea why this would be."

I found that this has nothing to do with gtk, but rather with the fact that my editor is running cargo while I'm trying to run it.

**Comment by gkoz** [comment]
@UrKr thanks for the update.

**Pull request merged**